src/Components/3DV/SceneView.tsx

```diff
@@ -372,10 +372,14 @@ function SceneView(props: ISceneViewProps, ref) {
                         );
 
                         camera.attachControl(canvas, false);
-                        camera.lowerRadiusLimit = 0;
                         cameraRef.current = camera;
                         cameraRef.current.zoomOn(meshes, true);
                         cameraRef.current.radius = radius;
+                        cameraRef.current.lowerRadiusLimit = 0;
```

Comment:
can we add a comment to the properties that help with the clipping so we remember?

Reply:
also a note on why *3 and *5?

Reply:
Added comments and made the values make a little more sense